example.pool.conf

```diff
@@ -38,7 +38,12 @@ log_level=info
 
 #### Cuckoo Config ####
 # edge bits (24)
+# WARNING, the 'edge_bits' works for cuckaroo ONLY at the current stage.
```

Comment:
the edg_bits use in too https://github.com/Qitmeer/qitmeer-miner/pull/112,so it useful for cuckoo

Reply:
fixed